resourcepolicies.md

```diff
@@ -0,0 +1,45 @@
+# Resource Policies Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/authz/resourcepolicies**   
+
+As we don't have yet an use case to iterate over all the existent resource policies the main endpoint is not implemented and a 405 error code is returned according to our [general error response codes](README.md#Error codes).
+
+Example: <http://dspace7.4science.it/dspace-spring-rest/#/dspace-spring-rest/api/core/collections>
+
+## Single Resource Policy
+**/api/authz/resourcepolicies/<:id>**
+
+Provide detailed information about a specific resource policy. The JSON response document is as follow
+```json
+{
+  "id": 2844,
+  "name": null,
+  "description": null,
+  "resourceType": "TYPE_SUBMISSION",
+  "action": "DEFAULT_BITSTREAM_READ",
```

Comment:
Shouldn't this correspond to the actions defined in DSpace API? E.g. `READ`, `WRITE`, `DELETE`, `ADD`, `ADMIN`... (https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/java/org/dspace/core/Constants.java#L75) ?

Reply:
I have changed the example in READ because it is the most common scenario. Yes, it is exactly what you point of, DEFAULT_BITSTREAM_READ is one of the option see https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/java/org/dspace/core/Constants.java#L115